### DISCUSSION TITLE
annotation tool built into birdnet-pi ?

### DISCUSSION THREAD

**Discussion by yoavfreund** [mcguirepr89/BirdNET-Pi#794] [Ideas]
It would be great if this already exists... 

An annotation tool that is part of the birdnet-pi, that way the raspberri's used to collect and automtically annotate bird songs can also be used to manually annotate and in this way provide trainining data for new types of birds.

Yoav Freund
UCSD
yfreund@ucsd.edu

**Comment by ehpersonal38** [comment]
Your best bet will be BirdWeather. I think I read that they have an annotation tool, and it's already integrated to be able to train future BirdNET models. The other best way to help provide training data is by uploading verified audio clips to eBird, Macaulay Library, and Xeno_Canto [(source)](https://github.com/kahst/BirdNET-Analyzer/issues/60#issuecomment-1439706149).
